packages/api/cms-api/src/public-upload/public-uploads.controller.ts

```diff
@@ -28,4 +28,17 @@ export class PublicUploadsController {
 
         return publicUploadsFile;
     }
+
+    @Get(":id")
+    @DisableCometGuards()
+    async getFileById(@Param("id") id: string): Promise<PublicUpload> {
+        return this.publicUploadsService.getFileById(id);
+    }
```

Comment:
Should this really be a REST endpoint? Why not GraphQL?

Reply:
What is the use case where you need this endpoint?

Is it secure enough?

Reply:
> Should this really be a REST endpoint? Why not GraphQL?

Since the file-upload is here I thought it makes sense to add this here as well. 
I'm not sure what the pros/cons for REST/GQL are in this case 🤷🏼‍♂️ 

> What is the use case where you need this endpoint?

This is needed for the admin file-upload field. 
When a file has been uploaded, only it's id is stored in the form-values. 
When showing the list of uploaded files, we need these values (file-name, file-size)

> Is it secure enough?

I have no idea, that's why I added @dkarnutsch as a reviewer 😅 
My logic was, since it's named "public uploads" it can all be public 🤷🏼‍♂️

Reply:
This endpoint has been removed.